testapp/build.gradle

```diff
@@ -36,13 +36,15 @@ android {
     def stagingManifestPlaceholders = [
             baseUrl        : property("MINIAPP_SERVER_BASE_URL") ?: "https://www.example.com/",
             isTestMode     : property("IS_TEST_MODE") ?: false,
+            hostAppInfo    : property("HOST_APP_INFO") ?: "MiniApp Demo App",
```

Comment:
A more analogous user agent info could be inferred from the existing user agent information for our sample app. What do you think?